### PR TITLE
Add option to pass schema as string, fixes #59

### LINKED ISSUE #59: Support for GraphQL IDL schema file
Feature suggestion: the linter, unlike graphql-tool’s `makeExecutableSchema`, relies not on a GraphQL IDL file, but instead on the much longer introspection query file.

I version control my schema with each client, so there test suites can be run against a static target, it _would be nice (tm)_ if I didn't need to do any legwork to convert my IDL to the JSON format.

### PR DESCRIPTION
**Pull request by christophercliff**
Attempted fix for #59. Could you please let me know if this is looks appropriate and what's expected in terms of test?

TODO:

- [x] Make sure all of the significant new logic is covered by tests
- [x] Rebase your changes on master so that they can be merged easily
- [x] Make sure all tests pass
- [x] Update CHANGELOG.md with your change
- [x] If this was a change that affects the external API, update the README

### CODE REVIEW

File: test/makeRule.js [hunk 1]
```diff
@@ -26,7 +28,7 @@ const parserOptions = {
 
 {
   const options = [
-    { schemaJson },
+    { schemaString },
```
Comment (jnwng) [inline test/makeRule.js]: i dont think you want to replace this — you should add a separate block of tests for this

File: src/index.js [hunk 2]
```diff
@@ -53,6 +54,9 @@ const defaultRuleProperties = {
   schemaJsonFilepath: {
     type: 'string',
   },
+  schemaString: {
```
Comment (jnwng) [inline src/index.js]: what do you think about using `typeDefs` instead? its a term i saw a lot in [graphql-tools](https://github.com/apollographql/graphql-tools) when referring to the schema when defined via the IDL

**Comment by jnwng** [comment]
@christophercliff beautiful, i was just working on this too! glad you got it all working. left a couple of comments, and do you mind filling out the Meteor CLA?

**Comment by jnwng** [comment]
also, do you mind updated the README accordingly?

File: src/index.js (hunk 2 above)
Comment (christophercliff) [inline src/index.js]: `makeExecutableSchema()` defines [`typeDefs`](http://dev.apollodata.com/tools/graphql-tools/generate-schema.html#makeExecutableSchema):

> typeDefs is a required argument and should be an array of GraphQL schema language strings or a function that takes no arguments and returns an array of GraphQL schema language strings. The order of the strings in the array is not important, but it must include a schema definition.

That's not exactly what's used here. Alternatively, [`buildSchema()`](http://graphql.org/graphql-js/utilities/#buildschema) just refers to it as `source`. [`printSchema()`](http://graphql.org/graphql-js/utilities/#printschema) refers to it as "schema in the Schema Language format".

I think `schemaString` is a decent compromise given the ambiguity and that the API is already using `schemaJson`.

**Comment by christophercliff** [comment]
@jnwng Please see updates. Could you give me an idea when this might be published?

**Comment by jnwng** [comment]
thank you @christophercliff! this should be available in `eslint-plugin-graphql@v1.1.0` (which is live)

**Pull request merged**